static/app/components/events/interfaces/crashContent/exception/sourceMapDebug.tsx

```diff
@@ -131,6 +132,33 @@ function getErrorMessage(
             : `https://docs.sentry.io/platforms/javascript/guides/${docPlatform}/sourcemaps/troubleshooting_js/#verify-artifact-names-match-stack-trace-frames`,
         },
       ];
+    case SourceMapProcessingIssueType.NO_URL_MATCH:
+      return [
+        {
+          title: t('Absolute Path Mismatch'),
+          desc: tct(
+            'The [absPath] of the stack frame doesn’t match any release artifact. Please refer to the instructions in our docs guide for help with troubleshooting the issue.',
```

Comment:
Should we show what the absPath of the stack frame is here? A little less useful than if there's a partial match but it might help?

Reply:
@roggenkemper I think it helps yea